tests/test_modeling.py

```
@@ -13,370 +13,325 @@
-            basic_setup['embed_dim']
+            basic_setup["batch_size"],
+            basic_setup["max_text_length"],
+            basic_setup["embed_dim"],
         )
-        assert mask.shape == (basic_setup['batch_size'], basic_setup['max_text_length'])
-    
+        assert mask.shape == (basic_setup["batch_size"], basic_setup["max_text_length"])
+
     def test_extracts_correct_embeddings(self, basic_setup):
         """Should place token embeddings at correct word positions."""
         words_embedding, mask = extract_word_embeddings(**basic_setup)
```

Ruff (0.15.1)

[warning] Unpacked variable `mask` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)